File: get_yt.js
const https = require('https');
https.get('https://www.youtube.com/@systemcarelaturlaptoprepair', (res) => {
  let data = '';
  res.on('data', chunk => data += chunk);
  res.on('end', () => {
    const matches = [...data.matchAll(/"videoId":"([^"]+)"/g)];
    const ids = [...new Set(matches.map(m => m[1]))];
    console.log("VIDEOS:", ids.slice(0, 5));
  });
});

File: get_yt.py
import urllib.request
import re

try:
    html = urllib.request.urlopen('https://www.youtube.com/@systemcarelaturlaptoprepair').read().decode('utf-8')
    videos = re.findall(r'"videoId":"([^"]+)"', html)
    seen = set()
    unique_videos = [x for x in videos if not (x in seen or seen.add(x))]
    print("VIDEOS:", unique_videos[:5])
except Exception as e:
    print("ERROR:", e)
